.github/workflows/perf.yml

```diff
@@ -43,6 +43,25 @@ jobs:
         run: |
           sudo heaptrack_print heaptrack_scap.data.zst -d ${{ steps.perf.outputs.heaptrack_scap }} &> heaptrack_scap_diff.txt
 
+      - name: Diff from master - gbench
```

Comment:
The algorithm is as follows:
* load all benchmarks present in master json
* for each of them, take the master `real_time` value
* for each of them, take the PR's head `real_time` value
* if PR's head does not have the benchmark, skip it
* compute the  diff between head and master
* compute the diff in percentage between head and master

Reply:
Have you ever considered to use the `compare.py` tool https://github.com/google/benchmark/blob/main/docs/tools.md#comparepy?

Reply:
Neat! I'll give it a look and eventually implement it!